### DISCUSSION TITLE
新的签名服务器有部署成功的吗

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#432] [一般讨论]
好懒，不知道Arm盒子能不能用以及有没有别的坑，比如有些说还得几天刷新devices.json的
顺便Tsuk1ko能不能推动点赞版那边跟一下新的go-cq版本(x

**Comment by Tsuk1ko** [comment]
催了（

签名服务器貌似目前尚未完善会导致封号，建议是先别用

**Reply by A2Sumie** [reply]
啧，唉

**Comment by Miuzarte** [comment]
那个签名服务器java写的，arm应该能跑，可以试试
至于封号，不管是gocq那边的discussion还是我自己实测，都是用了之后几个小时内会冻结一次，去解冻了再继续用就完全没问题了，用了几天到现在都没冻结过第二次

**Reply by Tsuk1ko** [reply]
那还行

**Reply by Miuzarte** [reply]
话说saucenao和a2d有没有什么镜像站反代站，bot在外头待了好久最近回国发现啥都连不上了（

**Reply by Tsuk1ko** [reply]
不清楚，不过你可以找一些白嫖的容器服务自己写个反代，vercel、glitch 、northflank 之类的

glitch 有冷启动要一分钟去访问一次保活

服务名字别带 proxy，不然被查到认为你违反 tos

**Comment by A2Sumie** [comment]
报告一下
1.1.0成功点起来，rk3568的弱鸡a55上会导致发送稍微有点慢
冻结一次，人脸+身份证校验那种立刻解除，目前2个号存活24h+无异常
